packages/web-components/src/card/card.template.ts

```diff
@@ -0,0 +1,30 @@
+import { ElementViewTemplate, html, when } from '@microsoft/fast-element';
+import type { Card } from './card.js';
+
+/**
+ * The template for the Fluent Card web-component.
+ * @public
+ */
+export function cardTemplate<T extends Card>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="group"
```

Comment:
I don't see `aria-selected` or `aria-disabled` as being supported for `role="group"`: https://w3c.github.io/aria/#group, what is the intent of that role as it relates to the card? This is something I'd probably expect to see in the spec.